addons/l10n_in/models/account_move_line.py

```diff
@@ -0,0 +1,9 @@
+from odoo import models
+from odoo import tools
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    def init(self):
+        tools.create_index(self._cr, 'account_move_line_move_product_index', self._table, ['move_id', 'product_id'])
```

Comment:
Do we still need this index?

Reply:
I m not sure about it, if it is not require we can delete it completely, anyways we got to add it again in fw-port since I guess we have `l10n_in_hsn_code` on move line